week2_day4/form.js

```diff
@@ -0,0 +1,36 @@
+function validateForm() {
+
+    var allLetters = /^[a-zA-Z]+$/;
+    var letter = /[a-zA-Z]/;
+    var number = /[0-9]/;
+
+    var firstName = document.order_form.first_name.value;
```

Comment:
Hmm... are you able to run this? Your html form does not have a surname or first name field, so it seems like this code would run with an error.